test/import/licence/fixtures/licences/agreements.js

```diff
@@ -0,0 +1,37 @@
+'use strict'
+
+module.exports = {
```

Comment:
Fixtures allow us to build a map of data from the import service to use across the import testing suite.